components/sidebar_right/sidebar_right.jsx

```diff
@@ -131,6 +125,19 @@ export default class SidebarRight extends React.Component {
     }
 
     render() {
+        const {
+            channel,
+            currentUser,
+            isFlaggedPosts,
+            isMentionSearch,
+            isPinnedPosts,
+            postRightVisible,
+            previousRhsState,
+            searchVisible,
+        } = this.props;
```

Comment:
I do appreciate the refactoring but I think it's a bit dangerous to do in the release branch. No action needed here, just something to think about for future release PRs